### ISSUE TITLE
Recognize invalid $schema field in lint

### ISSUE THREAD

**Issue by chuwy** [snowplow-incubator/igluctl#5]
It was working in 0.5.0, but we've got this regression in 0.6.0

**Comment by chuwy** [comment]
For self-describing schemas we have `$schema` keyword specifying an URL to the metaschema. The **only** one valid value is this:

https://github.com/snowplow/iglu-central/blob/master/schemas/com.amazon.aws.lambda/java_context/jsonschema/1-0-0#L2

But people tend to make typos there or just add some nonsense. Igluctl needs to recognize these invalid URIs (and it did before 0.6.0)

**Comment by istreeter** [comment]
Closing this because it is already fixed in the 0.7.x series:

```
FAILURE: Schema [com.acme/myschema/jsonschema/1-0-0] contains 1 following issues: 
1. Problem with $schema field: 
 - /     Given $schema is not http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#
```
